src/components/alert/__spec__.js

```diff
@@ -40,7 +34,7 @@ describe('Alert', () => {
     });
 
     describe("on internal elements", () => {
-      let wrapper = shallow(<Alert open={ true } title='Test' subtitle='Test' showCloseIcon={ true } />);
+      let wrapper = mount(<Alert open={ true } title='Test' subtitle='Test' showCloseIcon={ true } />);
```

Comment:
the Alert component inherits from the Dialog component.
Now that the title and subTitle are rendered via a Heading component, we need to mount it to get all the tags expected in the test.
Same goes for Confirm.